cli/package.json

```diff
@@ -10,12 +10,11 @@
   "dependencies": {
     "@identity.com/cryptid": "^0.1.1",
     "@identity.com/sol-did-client": "^2.0.0",
-    "@oclif/command": "^1",
-    "@oclif/config": "^1",
-    "@oclif/plugin-help": "^3",
+    "@oclif/core": "^1.5.0",
+    "@oclif/plugin-help": "^5",
```

Comment:
this is causing the build to break... if I set it as `^5.1.11` it works...

Is it good practice just to use `^5` instead of a specific version ? Also, my understanding is that `^5` should've pulled in the latest `5.1.11` yet it doesn't?